snap/snapcraft.yaml

```diff
@@ -192,6 +195,7 @@ parts:
       - libmcrypt-dev
       - libldap2-dev
       - libfreetype6-dev
+      - libreadline-dev
```

Comment:
We're not typically willing to introduce new dependencies into the snap. Finding documentation for how fulltextsearch works is difficult, but I'd be surprised if this was all that was required to enable all its providers. Have you actually built this snap and tested it with various providers?

Reply:
Yes, I have built the snap and upgraded my install with it. This has resolved the issue. I have the full text search app and the files & elastic search plug in.

https://apps.nextcloud.com/apps/fulltextsearch
https://apps.nextcloud.com/apps/files_fulltextsearch
https://apps.nextcloud.com/apps/fulltextsearch_elasticsearch